tests/php/modules/shortcodes/test_class.slideshow.php

```diff
@@ -24,4 +24,43 @@ public function test_shortcodes_slideshow() {
 		$this->assertNotEquals( $content, $shortcode_content );
 	}
 
+	public function test_shortcodes_slideshow_no_js() {
+		switch_to_blog( 104104364 ); // test.wordpress.com
```

Comment:
This can probably be abstracted into a `protected function setUp()` or `setUpBeforeClass()` and run only on wpcom environments.  In Jetpack, we should probably do something like manually creating db media item entries to use, instead of relying on an existing blog id.

https://phpunit.de/manual/current/en/fixtures.html